### PR TITLE
BaSP-MR-50: Class put and delete component 

### PR DESCRIPTION
**Pull request by josedaniele**
In this PR, the functionalities to update and delete a user were implemented along with their respective modals for success and error messages. The view utilized the GET request that my teammate developed in their ticket. Additionally, the mapping in this GET request was modified to render a component called "DataViewTd," which made data editing easier for me.

### CODE REVIEW

**Review by IgnacioGalceran — APPROVED** [review]
Everything works!

File: src/Components/Classes/List/list.module.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+tbody tr:nth-child(even) {
+  background-color: #EBEBEB;
+}
+
+tbody tr:nth-child(even):hover {
+  background-color: #c0b6b6;
+}
+
+.selected {
+  background-color: #373867 !important;
```
Comment (JuanCruzMoreira) [inline src/Components/Classes/List/list.module.css]: Delete all "!important"

**Review by JuanCruzMoreira — CHANGES_REQUESTED** [review]

File: src/Components/Classes/List/DataViewTd/DataViewTd.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
+import React from 'react';
```
Comment (JuanCruzMoreira) [inline src/Components/Classes/List/DataViewTd/DataViewTd.jsx]: Write both imports in the same line

File: src/Components/Classes/List/ClassList.jsx [hunk 3]
```diff
@@ -1,15 +1,24 @@
-import React, { useState } from 'react';
+import { useState } from 'react';
```
Comment (JuanCruzMoreira) [inline src/Components/Classes/List/ClassList.jsx]: Add React too

**Review by JuanCruzMoreira — CHANGES_REQUESTED** [review]

**Review by JuanCruzMoreira — APPROVED** [review]

**Pull request merged**